dependencies.props

```diff
@@ -157,7 +157,7 @@
     </StaticDependency>
 
     <StaticDependency Include="Microsoft.TargetingPack.Private.WinRT">
-      <Version>1.0.3</Version>
```

Comment:
I believe this guy won't get used at all, so we can probably just remove it. cc: @dagood